common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -146,4 +153,18 @@
             </ul>
         }
     </div>
+
+    <div class="menu-brand-extensions hide-until-desktop">
```

Comment:
I am not 100% Sure if this is the right way. I feel like this change requires some refactoring in the NewNavigation model instead.

I don't think it should wait for another PR to do it either. I think its necessary to make sure the campaign codes are correct per edition.